bootcamp/tutorials/quickstart/multimodal_rag_with_milvus.ipynb

```diff
@@ -0,0 +1,884 @@
+{
```

Comment:
<p>It is not a good option to git clone the whole project or use <code>cd</code> command in jupyter notebook. Remove this section and install necessary dependencies in the "Install Dependencies" section.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1398/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>